infra/core/ai/cognitiveservices.bicep

```diff
@@ -41,3 +44,5 @@ output id string = account.id
 output name string = account.name
 output skuName string = account.sku.name
 output key string = account.listKeys().key1
+output identityPrincipalId string = account.identity.principalId
```

Comment:
This should match the name of the other core files when they provide an id, which is just principalId